docs/security.html

```diff
@@ -1205,6 +1205,447 @@ <h4><a id="security_authz_cli" href="#security_authz_cli">Command Line Interface
         </tr>
     </tbody></table>
 
+    <h4><a id="security_authz_primitives" href="#security_authz_primitives">Privilege primitives</a></h4>
+    <p>Protocol calls are usually performing some operations on certain resources in Kafka. It is required to know the
+    operations and resources to set up an effective protection. In this section we'll list these operations and
```

Comment:
I'm not entirely certain on the correct grammar here, but `set up effective protection` sounds better to me